app/web/features/search/SearchResultsList.test.tsx

```diff
@@ -80,12 +88,20 @@ describe("SearchResultsList", () => {
   it("Is blank with no selection or query", () => {
     render(
       <SearchResultsList
-        updateMapBoundingBox={() => jest.fn()}
+        isLoading={false}
+        results={undefined}
+        error={undefined}
+        hasNext={undefined}
+        fetchNextPage={() => {}}
         selectedResult={undefined}
-        handleResultClick={mockHandleResultClick}
-        handleMapUserClick={mockHandleMapUserClick}
-        map={mockMapRef}
-        searchFilters={mockSearchFiltersFactory()}
+        setSelectedResult={() => {}}
```

Comment:
Do you need this empty function? If you're afraid of it erroring out I'd just add it as a default prop value within `SearchResultsList` itself.

Reply:
🤔🤔  not sure if I understand...